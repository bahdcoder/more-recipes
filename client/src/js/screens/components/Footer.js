import React from 'react';

export default class Footer extends React.Component {
  render() {
    return (
      <footer className="footer">
        <p>More-recipes app by bahdcoder @andela</p>
      </footer>
    );
  }
}
