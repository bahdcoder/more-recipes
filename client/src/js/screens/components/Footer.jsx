import React from 'react';

const Footer = () => ((
  <footer className="footer mt-5">
    <p>More-recipes app by bahdcoder @andela</p>
  </footer>
));

export default Footer;
