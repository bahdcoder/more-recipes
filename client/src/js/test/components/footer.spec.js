/* eslint-disable react/jsx-filename-extension */
import React from 'react';
import { shallow } from 'enzyme';
import setupTests from '../setupTests';
import Footer from '../../screens/components/Footer';

setupTests();

describe('The Footer component', () => {
  test('should mount without crashing', () => {
    const wrapper = shallow(<Footer />);
    expect(wrapper).toMatchSnapshot();
  });
});
