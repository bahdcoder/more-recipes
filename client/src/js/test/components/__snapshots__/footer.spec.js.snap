// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The Footer component should mount without crashing 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Footer />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <p>
        More-recipes app by bahdcoder @andela
      </p>,
      "className": "footer mt-5",
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": "More-recipes app by bahdcoder @andela",
      },
      "ref": null,
      "rendered": "More-recipes app by bahdcoder @andela",
      "type": "p",
    },
    "type": "footer",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <p>
          More-recipes app by bahdcoder @andela
        </p>,
        "className": "footer mt-5",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "More-recipes app by bahdcoder @andela",
        },
        "ref": null,
        "rendered": "More-recipes app by bahdcoder @andela",
        "type": "p",
      },
      "type": "footer",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
